metron-streaming/Metron-Topologies/src/main/resources/Metron_Configs/etc/env/config.properties

```diff
@@ -17,8 +17,8 @@
 
 ##### Kafka #####
 
-kafka.zk=zkpr1:2181,zkpr2:2181,zkpr3:2181
-kafka.broker=kfka1:6667
+kafka.zk=node1:2181
+kafka.broker=node1:6667
```

Comment:
I think these are going to break the common deploy. Shouldn't they remain zkpr[1-3] and kfka1?